src/fides/api/ops/models/policy.py

```diff
@@ -97,7 +99,7 @@ def _validate_rule(
             raise common_exceptions.RuleValidationError(
                 "Access Rules must have a storage destination."
             )
-    if action_type in [ActionType.consent.value, ActionType.update.value]:
+    if action_type in [ActionType.update.value]:
```

Comment:
We already technically had consent rules, we're just allowing ourselves to use them!